tests/benchmark/codes/manually_test_power_mnu.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
Labels have been fixed. I also fixed the call to the files for the NL power, because it wasn't finding them. However, the plots for the NL power have now changed (they look like linear power). Also, this code is not in the README nor in the wiki: https://github.com/LSSTDESC/CCL/wiki/Benchmarks. Pinging @c-d-leonard to see if we can fix these issues.

Reply:
I'm not even sure we need to have this code here. It looks like its contents could be ported to the benchmark notebook, it not already there. Do we use the plots produced here for the paper?